sdk/core/core-http/src/serializer.ts

```diff
@@ -6,12 +6,31 @@ import * as base64 from "./util/base64";
 import * as utils from "./util/utils";
 import { XML_ATTRKEY, XML_CHARKEY, SerializerOptions } from "./util/serializer.common";
 
+// This file contains an OpenAPI schema serializer, used by Autorest.Typescript.
```

Comment:
I would quibble with this wording, since that makes it sound like it can understand OpenAPI specifications, which it does not. Rather, it's a custom bit of code to serialize/deserialize according to `OperationSpec` objects generated by AutoRest.TypeScript from OpenAPI specifications.